services/content-publishing/package.json

```diff
@@ -5,9 +5,9 @@
   "main": "dist/apps/api/main.js",
   "scripts": {
     "build": "nest build api && nest build worker",
-    "build:swagger": "npx ts-node -r tsconfig-paths/register apps/api/src/build-openapi.ts",
-    "build:metadata": "npx ts-node apps/api/src/generate-metadata.ts",
-    "generate-swagger-ui": "npx --yes @redocly/cli build-docs swagger.json --output=./docs/index.html",
+    "build:swagger": "set -a ; . ./env.template ; npx ts-node -r tsconfig-paths/register apps/api/src/build-openapi.ts",
```

Comment:
Added `env.template` to each service openapi build because the build was failing without the environment variables.